webpack.config.ts

```diff
@@ -186,35 +192,18 @@ const plugins: webpack.WebpackPluginInstance[] = [];
 plugins.push(new CopyPlugin({
   patterns: [
     {
-      from: path.resolve(__dirname, 'node_modules/react/umd'),
-      to: path.resolve(__dirname, 'htdocs/vendor/js/react'),
+      from: `node_modules/react/umd/${
```

Comment:
i like this change!

Reply:
Yup, I had to look at the documentation to see if non-absolute paths were ok !